demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
+*/
+/* eslint-disable require-jsdoc */
+
+class ArticleRef {
+  constructor(lang, title, revision) {
+    this.lang = lang
+    this.title = title
+    this.revision = revision
+  }
+  fileName() {
+    return `${this.lang}.${this.title}.${this.revision}.json`
```

Comment:
Yes I believe we should. The reason is we may want to test 2 revisions of the same article.